addon/ng2/models/webpack-build-test.js

```diff
@@ -60,7 +60,7 @@ const getWebpackTestConfig = function(projectRoot, sourceDir) {
         { test: /\.css$/,  loaders: ['raw-loader', 'postcss-loader'] },
         { test: /\.styl$/, loaders: ['raw-loader', 'postcss-loader', 'stylus-loader'] },
         { test: /\.less$/, loaders: ['raw-loader', 'postcss-loader', 'less-loader'] },
-        { test: /\.scss$/, loaders: ['raw-loader', 'postcss-loader', 'sass-loader'] },
+        { test: /\.scss$|\.sass$/, loaders: ['raw-loader', 'postcss-loader', 'sass-loader'] },
```

Comment:
@hansl This is kinda sneaking a change not relative to the PR into this PR. It's a valid change but in the spirit of the PR, its not relevant. I don't think we want this.